backend/coins/eth/account.go

```diff
@@ -599,15 +599,23 @@ func (account *Account) newTx(args *accounts.TxProposalArgs) (*TxProposal, error
 			}
 		}
 	}
-	tx := types.NewTransaction(account.nextNonce,
-		*message.To,
-		message.Value, gasLimit, suggestedGasPrice, message.Data)
+	txData := &types.DynamicFeeTx{
```

Comment:
I think you can't just switch to this because it does not work with users that did not upgrade the BitBox02, right?

Can only use this if the firmware version is a at least 9.16.0 (or whatever version it ends up being released in)

Reply:
yeah, correct, thought of that too, how do you want to handle this if not a mandatory fw upgrade for 1559?

Reply:
use eip1559 only if fw is at least 9.16.0, otherwise use legacy.